src/main/java/mekanism/common/content/network/transmitter/LogisticalTransporterBase.java

```diff
@@ -403,6 +405,14 @@ public <BE extends BlockEntity & IAdvancedTransportEjector> TransitResponse inse
         return insert(outputter, outputterPos, request, color, doEmit, min);
     }
 
+    public TransitResponse insertMaybeRR(@NotNull TileEntityConfigurableMachine outputter, BlockPos outputterPos, TransitRequest request, @Nullable EnumColor color, boolean doEmit, int min) {
```

Comment:
Instead of duplicating methods like this, would it maybe make more sense to have `TileEntityConfigurableMachine` just implement `IAdvancedTransportEjector` and wrap the calls to the implementation on `TileComponentEjector`?

Reply:
Main reason I didn't is that it avoids the custom send home methods being needed

But that could be implemented